apps/www/registry/default/ui/spinner.tsx

```diff
@@ -0,0 +1,36 @@
+import React from "react"
+
+import { cn } from "@/lib/utils"
+
+const Spinner = React.forwardRef<
+  HTMLDivElement,
+  React.ComponentPropsWithoutRef<"div">
+>(({ className, ...props }, ref) => {
+  const computeDelay = (i: number) => `${-1.2 + i * 0.1}s`
+  const computeRotation = (i: number) => `${i * 30}deg`
+  return (
+    <div
+      className={cn("size-5", className)}
+      role="status"
+      aria-label="Loading"
+      ref={ref}
+      {...props}
+    >
+      <div className="relative left-1/2 top-1/2 size-full">
+        {[...Array(12)].map((_, i) => (
```

Comment:
creating 12 divs inline seems like a funky way of achieving this, no?

Reply:
Yeah i'll admit it, that this doesn't seem like good practice, but to be fair this is how I saw most spinners from other design systems being implemented

Reply:
@cmawhorter any suggestions on how you would implement this?